react-app/src/Home/index.js

```diff
@@ -15,7 +16,7 @@ export default function Home() {
       <NavBar setShowMenu={setShowMenu} navbarColor='#FFB300' />
       {showMenu && <SideMenu setShowMenu={setShowMenu} />}
       <View style={styles.greeting}>
-        <Text style={styles.greetingText}>Good Morning</Text>
+        <Greeting style={styles.greeting} />
```

Comment:
I think `style={styles.greeting}` is duplicated in line 18 and line 19, the style in line 19 is unnecessary and useless.